src/Quadrature/quadrature.jl

```diff
@@ -96,10 +108,16 @@ end
 for dim in 2:3
     @eval begin
         function QuadratureRule{RefSimplex{$dim}}(::Type{T}, quad_type::Symbol, order::Int) where T
-            if $dim == 2 && quad_type === :legendre
-                data = _get_gauss_tridata(order)
-            elseif $dim == 3 && quad_type === :legendre
-                data = _get_gauss_tetdata(order)
+            if $dim == 2 && quad_type === :dunavant
+                data = _get_dunavant_gauss_tridata(order)
+            elseif $dim == 2 && quad_type === :gaussjacobi
+                data = _get_gaussjacobi_tridata(order)
+            elseif $dim == 3 && quad_type === :jinyun
+                data = _get_jinyun_tet_quadrature_data(order)
```

Comment:
I think we should delete this option, since the current implementation is "keast" and we don't need to implement all possible quadrature rules. Of course, in the docs, we should acknowledge that `keast` orders 2 and 3 comes from jinyun.

Reply:
I would leave it for reference, because for me it was not that straight forward to figure out how to get the rules for order 1..3 from the Keast paper. This way it is directly obvious.

Reply:
IMO, we should make this clear from the docs rather than adding more complexity/options to the code.